### PR TITLE
feat(route): add WFDF

### PR DESCRIPTION
**Pull request by HankChow**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/wfdf/news
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/en/other.md [hunk 1]
```diff
@@ -311,6 +311,12 @@ Board and Build can be found in [here](http://api.ineal.me/tss/status)
 
 <RouteEn author="TonyRL" example="/urbandictionary/random" path="/urbandictionary/random" radar="1" rssbud="1"/>
 
+## WFDF
+
+### News
+
+<Route author="HankChow" example="/wfdf/news" path="/wfdf/news"/>
```
Comment (TonyRL) [inline docs/en/other.md:318]:
```suggestion
<RouteEn author="HankChow" example="/wfdf/news" path="/wfdf/news" radar="1"/>
```

File: docs/other.md [hunk 2]
```diff
@@ -367,6 +367,12 @@ type 为 all 时，category 参数不支持 cost 和 free
 
 </Route>
 
+## WFDF
+
+### 新闻
+
+<Route author="HankChow" example="/wfdf/news" path="/wfdf/news"/>
```
Comment (TonyRL) [inline docs/other.md:374]:
```suggestion
<Route author="HankChow" example="/wfdf/news" path="/wfdf/news" radar="1"/>
```

File: lib/v2/wfdf/news.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        guid: item.guid.rendered,
+        description: item.content.rendered,
+        pubDate: parseDate(item.date_gmt),
+        author: item._embedded.author[0].name,
+    }));
+
+    ctx.state.data = {
+        title: 'WFDF News',
+        link: `${baseUrl}/news/`,
+        image: `${baseUrl}/wfdf/favicon.ico`,
```
Comment (TonyRL) [inline lib/v2/wfdf/news.js:25]: This URL does not exist.

File: lib/v2/wfdf/radar.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'wfdf.sport': {
+        _name: 'WFDF',
+        '.': [
+            {
+                title: 'WFDF News',
```
Comment (TonyRL) [inline lib/v2/wfdf/radar.js:6]:
```suggestion
                title: 'News',
```

File: docs/en/other.md (hunk 1 above)
Comment (HankChow) [reply docs/en/other.md]: updated

File: lib/v2/wfdf/radar.js (hunk 4 above)
Comment (HankChow) [reply lib/v2/wfdf/radar.js]: updated

File: lib/v2/wfdf/news.js (hunk 3 above)
Comment (HankChow) [reply lib/v2/wfdf/news.js]: updated

File: docs/other.md (hunk 2 above)
Comment (HankChow) [reply docs/other.md]: updated

**Pull request merged**